### DISCUSSION TITLE
What else do we need before we can deploy /web to prod?

### DISCUSSION THREAD

**Discussion by jamesgeddes** [GeekZoneHQ/web#331] [General]
Please either,
- provide your approval for /web to be deployed to production
- explain what else is needed before you will be happy to provide your approval

**Comment by jamesgeddes** [comment]
@GeekZoneHQ/q Please discuss.

**Comment by tbentham** [comment]
I'm fine with the front-end. Having said that, if anyone else has noticed anything that looks odd, please say!

**Reply by Freshrojek** [reply]
I think the front-end looks good
